### ISSUE TITLE
About the ensemble model

### ISSUE THREAD

**Issue by tuji-sjp** [kazuto1011/grad-cam-pytorch#26]
Hello, I fused the logits of the three models. Maybe you know how does this ensemble model gets its GradCAM?

**Comment by kazuto1011** [comment]
You can backpropagate score gradients to each model and get three Grad-CAM heatmaps. Unfortunately, this code only supports a single heatmap.
